Source/WebCore/platform/ios/PlatformPasteboardIOS.mm

```diff
@@ -46,6 +46,7 @@
 #import <wtf/RuntimeApplicationChecks.h>
 #import <wtf/URL.h>
 #import <wtf/cocoa/NSURLExtras.h>
+#import <wtf/cocoa/RuntimeApplicationChecksCocoa.h>
```

Comment:
This change seems unrelated.  Is it necessary to fix unified builds after this?

Reply:
Yes.